%\VignetteKeywords{Database}
%\VignetteDepends{prostateCancerGrasso}
%\VignettePackage{prostateCancerGrasso}
%\VignetteIndexEntry{prostateCancerGrasso}

\documentclass{article}

\begin{document}
\SweaveOpts{concordance=TRUE}

\emph{The Mutational Landscape of Lethal Castrate Resistant Prostate Cancer}
\emph{Grasso et al. (2012) doi:10.1038/nature11125}

\begin{quote}
Characterization of the prostate cancer transcriptome and genome has identified chromosomal rearrangements and copy number gains/losses, including ETS gene fusions, PTEN loss and androgen receptor (AR) amplification, that drive prostate cancer development and progression to lethal, metastatic castrate resistant prostate cancer (CRPC)1. As less is known about the role of mutations2-4, here we sequenced the exomes of 50 lethal, heavily-pretreated metastatic CRPCs obtained at rapid autopsy (including three different foci from the same patient) and 11 treatment naïve, high-grade localized prostate cancers. We identified low overall mutation rates even in heavily treated CRPC (2.00/Mb) and confirmed the monoclonal origin of lethal CRPC. Integrating exome copy number analysis identified disruptions of CHD1, which define a subtype of ETS‑ prostate cancer. Similarly, we demonstrate that ETS2, which is deleted in ~1/3 of CRPCs (commonly through TMPRSS2:ERG fusions), is a prostate cancer tumor suppressor that can also be deregulated through mutation. Further, we identified recurrent mutations in multiple chromatin/histone modifying genes, including MLL2 (mutated in 8.6\% of prostate cancers), and demonstrate interaction of the MLL complex with AR, which is required for AR mediated signaling. We also identified novel recurrent mutations in the AR collaborating factor FOXA1, which is mutated in 5 of 147 (3.4\%) prostate cancers (both untreated localized prostate cancer and CRPC) , and showed that mutated FOXA1 represses androgen signaling and increases tumor growth in vitro and in vivo. Proteins that physically interact with AR, such as the ERG gene fusion product, FOXA1, MLL2, UTX, and ASXL1 were found to be mutated in CRPC, suggesting novel drivers of prostate cancer progression and potential resistance mechanisms to anti-androgen therapies. In summary, we describe the mutational landscape of a heavily treated metastatic cancer, identify novel mechanisms of AR signaling deregulated in prostate cancer, and prioritize candidates for future study.
\end{quote}

In this document, I describe how the Agilent expression data for this datast was processed and saved into an object for analysis in Bioconductor. First of all load the relevant libraries for grabbing and manipulaing the data

<<eval=FALSE>>=
library(GEOquery)
@


Now use the `getGEO` function with the correct ID

<<eval=FALSE>>=
geoData <- getGEO("GSE35988")
geoData
@

We notice that the samples are spread across two different datasets, each of which is a different list entry in the object we just created. We create a new pheno data object that has the sample names and analysis groups. We also cut-down the feature matrix to just ID, Entrez and Symbol.

<<eval=FALSE>>=

pd <- rbind(pData(geoData[[1]]),pData(geoData[[2]]))

groups = NULL
groups[grep('benign', pd$characteristics_ch2, perl=TRUE)] = "Benign"
groups[grep('localized', pd$characteristics_ch2, perl=TRUE)] = "HormomeDependant"
groups[grep('metastatic', pd$characteristics_ch2, perl=TRUE)] = "CastrateResistant"

pd <- data.frame(geo_accession=pd$geo_accession, Sample.Name= pd$title,Group=groups)

fd <- fData(geoData[[1]])[,c("ID","GENE","GENE_SYMBOL")]

exp <- cbind(exprs(geoData[[1]]), exprs(geoData[[2]]))

geoData <- geoData[[1]]
exprs(geoData) <- exp
pData(geoData) <- pd
fData(geoData) <- fd

grasso <- geoData
save(grasso, file="data/grasso.rda",compress="xz")
@


\end{document}